.env.example

```diff
@@ -0,0 +1,2 @@
+PREFIX=
```

Comment:
do you think the prefix should be an environment variable? I think it should be constant regardless of what environment the app is running on

Reply:
Ah, that's because everyone is working on this. Which means everyone will bring their development version to server. So, to prevent command clash, this was the only easy fix.

Reply:
I believe we should all agree on a prefix. I like `!` but anything goes.

Reply:
Fixed. You can either set the `PREFIX` or not. If not set, the app will fallback to `!` as it's prefix